creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -189,112 +196,6 @@ class SnapbackSM {
     }
   }
 
-  /**
```

Comment:
all of this code here is moved to the peerSetManager.js file

Reply:
this logic is instead replaced with helper methods from peerSetManager.js + try/catches, and are now in the 418-435